src/Symfony/Component/Config/Tests/Resource/DirectoryResourceTest.php

```diff
@@ -96,8 +96,10 @@ public function testIsFreshNewFileWithDifferentPattern()
     public function testIsFreshDeleteFile()
     {
         $resource = new DirectoryResource($this->directory);
+        $time = time();
+        sleep(1);
         unlink($this->directory.'/tmp.xml');
-        $this->assertFalse($resource->isFresh(time()), '->isFresh() returns false if an existing file is removed');
+        $this->assertFalse($resource->isFresh($time), '->isFresh() returns false if an existing file is removed');
```

Comment:
Good catch; I didn't know about that convention. I only see that group at the class-level. Is it okay to use it at the test-level?